tests/performance/test_auth_methods.py

```diff
@@ -0,0 +1,132 @@
+"""
+Results:
+
+| method        | project_size | reuse_connections | unsafe_skip_rsa_key_validation | duration |
+|---------------|--------------|-------------------|--------------------------------|----------|
+| User Password |        1,000 | False             | -                              |  234.09s |
+| User Password |        1,000 | True              | -                              |   78.34s |
+| Key Pair      |        1,000 | False             | False                          |  271.47s |
```

Comment:
Have these stats been updated since yesterday’s key-pair caching PR?

If not, probably worth updating as a favor to our future selves 🙂

Reply:
Yeah, I reran them after the caching. I didn't rerun the user/password ones since they shouldn't be affected.